admin/src/templates/header.html

```diff
@@ -0,0 +1,24 @@
+<nav class="header">
+    <div class="container">
+      <div class="navbar-header">
+        <h1 translate>admin.app.name</h1>
+      </div>
+      <div class="extras">
+        <span class="dropdown options">
+          <a data-toggle="dropdown">
+            <div class="mm-icon mm-icon-inverse">
+              <span class="fa fa-fw fa-bars"></span>
+            </div>
+          </a>
+          <ul class="dropdown-menu mm-dropdown-menu">
+            <li>
+              <a href="#" ng-click="logout()">
+                <i class="fa fa-fw fa-power-off"></i>
+                <span translate>Log Out</span>
+              </a>
+            </li>
+          </ul>
```

Comment:
I don't think we should use a dropdown for a single option - we can add the dropdown later if we want to add more items here. For now just include the "logout" button where the fa-bars icon is.